contrib/add_license_to_files.jl

```diff
@@ -58,7 +58,7 @@ const skipfiles = [
     "../src/support/tzfile.h",
     "../src/support/utf8.c",
     "../test/perf/micro/randmtzig.c",
-    "../src/support/crc32c.c",
+    "../src/crc32c.c",
```

Comment:
Did you forget to do a `git mv` for this file?  The diff is confusing because it as if you removed the file and added it elsewhere.